packages/react/ds/src/footer/footer.tsx

```diff
@@ -1,85 +1,63 @@
+import { ReactNode } from 'react';
 import GovieLogoHarpWithText from '../assets/logos/gov-of-ireland/harp-gold-text-green.js';
-import { Container } from '../container/container.js';
-import Anchor from '../primitives/anchor.js';
 import { SectionBreak } from '../section-break/section-break.js';
 
-type FooterLink = {
-  label: string;
-  href: string;
-  external?: boolean;
-};
-
 export type FooterProps = {
-  links?: FooterLink[];
-  secondaryNavLinks?: {
-    heading: string;
-    links: FooterLink[];
-  }[];
+  primarySlot?: ReactNode;
+  secondarySlot?: ReactNode;
+  utilitySlot?: ReactNode;
   dataTestid?: string;
+  logoComponent?: ReactNode;
+  className?: string;
 };
 
-export function Footer({ links, secondaryNavLinks, dataTestid }: FooterProps) {
+export function Footer({
+  primarySlot,
+  secondarySlot,
+  utilitySlot,
+  dataTestid = 'govie-footer',
+  logoComponent = <GovieLogoHarpWithText />,
+  className = '',
+}: FooterProps) {
```

Comment:
if you extend `FooterProps` with `& React.HTMLAttributes<HTMLDivElement>` you don't need to define and propagate `ARIA` and `data-...` props